### PR TITLE
Tevor rbsmfixup

### PR DESCRIPTION
**Pull request by TrevorDecker**

### CODE REVIEW

**Comment by TrevorDecker** [comment]
@msebek  can you review this

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,6 @@ surface_src/java_src/Alice/src/com/roboclub/robobuggy/main/config.java
 real_time/arduino_src/radio_buggy_mega/fingerprint.h
 
 real_time/surface_src/java_src/Alice/.classpath
+
+#eclipse's nonsense
```
Comment (msebek) [inline .gitignore]: Add a space here, so it's like the rest of the comments?

# eclipse generated files
real_time/surface_src/java_src/Alice/.classpath
real_time/surface_src/java_src/.recommenders/
**/.metadata/*

That last one should keep all .metadata out, no matter where they end up in the tree. Maybe the same should go for .recommenders, if that's something eclipse decides where to put based on workspace?

Add a comment about fingerprint for good hygiene?

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java [hunk 2]
```diff
@@ -84,7 +97,8 @@ public int matchDataMinSize() {
 
 	@Override
 	public int baudRate() {
-		return 9600;
+		return 115200;
+		//return 9600;
 	}
```
Comment (msebek) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java]: wat? this is changing for a good reason, right?

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java [hunk 3]
```diff
@@ -38,13 +38,19 @@
...
 	public static boolean ENCODER_DEFAULT = true;
 	public static boolean VISION_SYSTEM_DEFAULT = true;
 	
+	//sensor com ports
+	public static String COM_PORT_IMU = "COM6";
+	public static String COM_PORT_GPS_INTEGRATED = "COM5";
+	public static String COM_PORT_GPS_STANDALONE = "COM8";
+	public static String COM_PORT_ENCODER = "COM3";
+	public static String COM_PORT_DRIVE_CONTROL = "COM9";
+
```
Comment (msebek) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java]: maybe it's worth having a note that these lines should probably be reverted before new commits, so as not to ruin other people's days?

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java (hunk 2 above)
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java]: Yes, we want to get data at faster rate for the arduino.  Really frees up the arduinos time and gets us many more messages per second.

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java [hunk 4]
```diff
@@ -33,27 +37,36 @@
...
-	Publisher messagePub;
-	Publisher statePub;
+	Publisher messagePub_enc;
+	Publisher messagePub_pot;
+	Publisher statePub_enc;
+	Publisher statePub_pot;
+
 
-	public EncoderNode(SensorChannel sensor) {
+	public EncoderNode(SensorChannel sensor_enc,SensorChannel sensor_pot) {
```
Comment (msebek) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java]: should there be a space here?

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java (hunk 3 above)
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java]: I would rather have the versions for the surface in the repo.  Eventually we will have the values come from a gitignored config file

**Pull request merged**